docs/source/contributing/documentation.rst

```diff
@@ -318,6 +318,25 @@ elements that are either a ``str`` or ``None``;
 ``(str, int)``; ``Tuple[:class:`int`, ...]`` for a tuple of variable
 length with only integers.
 
+.. note::
+   As a helper for tool for typesets, you can use ``typestring-parser`` which can 
+   be accessed by first downloading it via ``pip`` - ``pip install typestring-parser`` and
+   then using ``from typestring_parser import parse``.
+
+   Example
+   .. testsetup::
```

Comment:
You just want a code block, right? Then this should do. Indentation of the block content seems to be missing though.

```suggestion
   Example::
```

Reply:
Ok, thank you.